todo_app/app.py

```diff
@@ -1,11 +1,20 @@
-from flask import Flask
+from crypt import methods
+from flask import Flask, redirect, render_template, request
+from todo_app.data.session_items import add_item, get_items
 
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
 @app.route('/')
 def index():
-    return 'Hello World!'
+    #return 'Hello World!!!!'
```

Comment:
I'm a big fan of deleting code rather than commenting it out - as long as it was in a past Git commit then we can always recover it anyway, and removing it removes the mental overhead of having to read past it/maintain it